src/TogglePassword/doc/index.rst

```diff
@@ -259,6 +259,35 @@ Then in your form, add your controller as an HTML attribute::
         // ...
     }
 
+Usage without Symfony Forms
+---------------------------
+
+You can also use the TogglePassword with native HTML inputs.
+It requires a little bit more work but it's still possible.
+Here is an example::
+
+    // ...
+
+    <label for="password">Password</label>
+    <input
+        id="password"
+        name="password"
+        type="password"
+        autocomplete="current-password"
+        required
+        placeholder="Password"
+        {{ stimulus_controller('@symfony/ux-toggle-password--toggle-password', {
+                visibleIcon: 'Default', // Here you can pass an SVG tag or a string. Default is the value for the default svg
+                visibleLabel: 'Show', // This field is optional
+                hiddenIcon: 'Default', // Here you can pass an SVG tag or a string. Default is the value for the default svg
```

Comment:
For `visibleIcon` and `hiddenIcon`, can you update the `controller.ts` file itself so that `Default` is the default? It shouldn't be necessary to specify `Default` manually here.

Reply:
Yes, I'll do this.